### ISSUE TITLE
kops 1.5.0-alpha3/alpha4: docker no longer able to download images

### ISSUE THREAD

**Issue by koendc** [kubernetes/kops#1542]
I'm updating a cluster using kops 1.5.0-alpha3. I was using an older version (git-6cbe5b4) before, but we want to use r4 instances. Since then, on new nodes docker is no longer able to download images. 

kops version: 1.5.0-alpha3
release channel: stable
image: kope.io/k8s-1.4-debian-jessie-amd64-hvm-ebs-2016-10-21

This is the -hopefullly relevant- docker log excerpt:
`
Jan 18 14:03:06 ip-172-28-0-159 docker[1235]: protokube version 0.1
Jan 18 14:03:06 ip-172-28-0-159 docker[1235]: I0118 14:03:06.749446       1 aws_volume.go:71] AWS API Request: ec2metadata/GetMetadata
Jan 18 14:03:06 ip-172-28-0-159 docker[1235]: I0118 14:03:06.750697       1 aws_volume.go:71] AWS API Request: ec2metadata/GetMetadata
Jan 18 14:03:06 ip-172-28-0-159 docker[1235]: I0118 14:03:06.751259       1 aws_volume.go:71] AWS API Request: ec2metadata/GetMetadata
Jan 18 14:03:06 ip-172-28-0-159 docker[1235]: I0118 14:03:06.753662       1 aws_volume.go:71] AWS API Request: ec2/DescribeInstances
Jan 18 14:03:07 ip-172-28-0-159 docker[1235]: I0118 14:03:07.046000       1 main.go:95] Setting cluster-id from cloud: kubernetes-kops.cpp-dev-po.amiefarm.com
Jan 18 14:03:07 ip-172-28-0-159 docker[1235]: I0118 14:03:07.046025       1 main.go:102] Setting dns-internal-suffix to ".internal.kubernetes-kops.cpp-dev-po.amiefarm.com"
Jan 18 14:03:07 ip-172-28-0-159 docker[1235]: I0118 14:03:07.046333       1 dnscontroller.go:103] starting DNS controller
Jan 18 14:03:13 ip-172-28-0-159 docker[1123]: time="2017-01-18T14:03:13.068405984Z" level=error msg="Handler for GET /images/gcr.io/google_containers/pause-amd64:3.0/json returned error: No such image: gcr.io/google_containers/pause-amd64:3.0"
`

It seems like```/lib/systemd/system/docker.service``` is no longer updated:

Using protokube 1.4:
```[Service]
Type=notify
EnvironmentFile=/etc/sysconfig/docker
ExecStart=/usr/bin/docker daemon -H fd:// "$DOCKER_OPTS"
```

With protokube 1.5.0-alpha3:
```[Service]
Type=notify
# the default is not to use systemd for cgroups because the delegate issues still
# exists and systemd currently does not support the cgroup feature set required
# for containers run by docker
ExecStart=/usr/bin/docker daemon -H fd://
```

**Comment by koendc** [comment]
I just checked with kops-1.5.0-alpha4: the issue is still there.

**Comment by justinsb** [comment]
@koendc thanks so much - you're 100% correct.  Fix is in #1702, but now I'm trying to figure out how this wasn't more visible.  I suspect you might have a second issue - I was able to reproduce the service update failure, but not the underlying problem (i.e. my instances still came up OK!)
